source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2406,12 +2430,15 @@ Value::ValueType SwiftLanguageRuntime::GetValueType(
     }
 
     if (static_type_flags.AllSet(eTypeIsSwift | eTypeIsPointer) &&
-        static_type_flags.AllClear(eTypeIsArchetype)) {
+        static_type_flags.AllClear(eTypeIsGenericTypeParam)) {
+      // FIXME: This branch is not covered by any testcases in the test suite.
```

Comment:
Then should we remove it or add a test?

Reply:
Yes. This is just something I noticed while working on this patch. I'll put an unreachable there and see if it sticks.